app/services/request_key_manager.rb

```diff
@@ -1,15 +1,27 @@
 class RequestKeyManager
   cattr_accessor :private_key do
-    OpenSSL::PKey::RSA.new(
-      File.read(Rails.root.join('keys', 'saml.key.enc')),
-      Figaro.env.saml_passphrase
-    )
+    key_file = Rails.root.join('keys', 'saml.key.enc')
+    begin
+      OpenSSL::PKey::RSA.new(
+        File.read(key_file),
+        Figaro.env.saml_passphrase
+      )
+    rescue OpenSSL::PKey::RSAError
+      Rails.logger.error("Failed to load #{key_file.inspect}. Bad passphrase?")
+      raise
+    end
   end
 
   cattr_accessor :equifax_ssh_key do
-    OpenSSL::PKey::RSA.new(
-      File.read(Rails.root.join('keys', 'equifax_rsa')),
-      Figaro.env.equifax_ssh_passphrase
-    )
+    key_file = Rails.root.join('keys', 'equifax_rsa')
+    begin
+      OpenSSL::PKey::RSA.new(
+        File.read(key_file),
+        Figaro.env.equifax_ssh_passphrase
+      )
+    rescue OpenSSL::PKey::RSAError
+      Rails.logger.error("Failed to load #{key_file.inspect}. Bad passphrase?")
+      raise
```

Comment:
should we re-raise a different error that points here? or update the error message?